modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You installed the OpenShift CLI (`oc`).
+* Your oc client must be updated to at least verion 4.13.0. 
+* Your {product-title} cluster must be installed on either the AWS or Azure platform. 
++
+For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_.
+
+
+.Procedure
+. Verify `RetrievedUpdates=True` in Cluster Version Operator (CVO) conditions by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade
```

Comment:
@shellyyang1989 can you please confirm if we see RetrievedUpdates=True in `oc adm upgrade` outout. AFAIR we do not.

Reply:
Right, we can't see RetrievedUpdates=True in `oc adm upgrade` outout. We can get it by 
`oc get clusterversion/version -o=jsonpath="{.status.conditions[?(.type=='RetrievedUpdates')].status}"`

Reply:
Updating! ty :)